Automattic/wp-calypso
# Jetpack Pricing Page: Make Jetpack CRM Monthly product disabled.

### Changes proposed in this Pull Request

We only sell Jetpack CRM yearly plans so we need to make the CRM Monthly option disabled.

This PR makes the Jetpack CRM Monthly product card disabled (The card is greyed out, purchase button disabled, and displays a message "Only available in yearly billing".

This PR is an enhancement of https://github.com/Automattic/wp-calypso/pull/48678 which completely removes the Jetpack CRM **_monthly_** product from being displayed when the monthly billing toggle is selected, however it was discussed that it would be better UX to have the _monthly_ CRM product greyed out and disabled instead of having it just disappear on them when they toggle to monthly billing.

### Testing instructions

* Checkout and run calypso and jetpack cloud environments concurrently. (`yarn start` and `yarn start-jetpack-cloud-p`) OR you can just run the Calypso Live link
* Go to `calypso.localhost:3000/plans/:site`.
*  Select the "Billing Monthly" toggle and verify that the `CRM Entrepreneur` product is greyed out, the purchase button is disabled, and it displays a message "Only available in yearly billing".
(See screen capture below)
* Go to `jetpack.cloud.localhost:3001/plans` and verify the same behavior above.


![chrome-capture (19)](https://user-images.githubusercontent.com/11078128/103799839-a0f8da80-5019-11eb-8d66-4af895dd6951.gif)


Fixes 1164141197617539-as-1199720441821493
